demo-terraform/aws/wordpress_ec2_alb_rds/alb.tf

```diff
@@ -1,53 +1,58 @@
 locals {
-    lb_name = "alb${random_string.lab_id.result}"
-    tg_name = "tg${random_string.lab_id.result}"
+  lb_name = "alb${random_string.lab_id.result}"
+  tg_name = "tg${random_string.lab_id.result}"
 }
 
 resource "aws_lb" "web" {
```

Comment:
**Accurics remediation**
 
**[MEDIUM]: Ensure access logging is enabled for AWS LB (Load Balancer)**
 
**Vulnerability**
AWS classic load balancers do not have access logging enabled.Access logging should be enabled in order to analyze statistics, diagnose issues, and retain data for regulatory or legal purposes.
 
**Remediation**
Access logging is an optional feature of Elastic Load Balancing that is disabled by default. After you enable access logging for your load balancer, Elastic Load Balancing captures the logs and stores them in the Amazon S3 bucket that you specify as compressed files. You can disable access logging at any time.

source: accurics-